benchmark_runner/grafana/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/centos/centos:stream9
+
+WORKDIR /app
+
+ENV TERRAFORM_VERSION=1.5.4
+
+RUN dnf update -y && \
+    dnf install -y git go wget unzip && \
+    go install github.com/google/go-jsonnet/cmd/jsonnet@latest && \
+    go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest && \
+    wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    mv terraform /usr/local/bin/ && \
+    rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+
+ENV PATH="/root/go/bin:${PATH}"
+
+CMD ["/bin/sh", "-c", "terraform init; make > dashboard.txt;  python3 extract_dashboard.py dashboard.txt dashboard.json; terraform destroy -auto-approve > /dev/null 2>&1"]
```

Comment:
I think there's a much easier way you can do this:

```
CMD ["/bin/sh", "-c", "terraform init; make |/bin/sh > dashboard.json; terraform destroy -auto-approve > /dev/null 2>&1"]
```
and then you won't need extract_dashboard.py at all.

Reply:
Tried, but its not working:

```
[root@6e8e49594c69 app]# make |/bin/sh > dashboard.json
/bin/sh: line 1: $'\E[0m\E[1mdata.jsonnet_file.dashboard:': command not found
```

Reply:
I have updated the extract code for suffix: **\nEOT\n**
`r'dashboard = <<EOT(.*?)\nEOT\n'`
That should solve the EOT issue.

Reply:
I think you need to run `terraform init --no-color`.

Reply:
From https://developer.hashicorp.com/terraform/cli/commands/init
```
-no-color Disable color codes in the command output.
```

You're much better off not trying to fight what terraform init is trying to do here.

Reply:
What exactly does `make` produce here?  Spaces between the variable name, the `=`, and the text are not valid shell, so maybe it's not producing what I think it is and that regex may be the only way to go.  But I'd like to see both the `Makefile` and the output.

Reply:
The `terraform init` is not related to the problem at all.
The problem is with the `terraform apply` inside Makefile.
This command converts jsonnet(grafonnet) code into json and its using [Jsonnet terraform rendered code](https://registry.terraform.io/providers/alxrem/jsonnet/latest/docs) that rendered it into constant output
```
dashboard = <<EOT
{
   "description": "Dashboard for Faro",
   "graphTooltip": "aaa",
   "title": "Faro dashboard"
}

EOT
```

Reply:
```
.PHONY: apply
apply:
        @cd jsonnet/ && \
                jb install && \
                cd ../ && \
                terraform apply -auto-approve > dashboard.txt && \
                python3 extract_dashboard.py dashboard.txt dashboard.json
```
@RobertKrawitz, I put all inside Makefile, WDYT ?

Reply:
Might as well put it all in the makefile; I still think you should avoid that temporary file (dashboard.txt).

Reply:
Where does `/root/grafonnet` come from?

Reply:
terraform apply -auto-approve -json => generates corrupted json output

Reply:
/root/grafonnet/ => You should put your local path.

Reply:
Local path to what?

Can you include the generated JSON?

Reply:
[dashboard.json](https://github.com/redhat-performance/benchmark-runner/blob/3ec88b4167cca890904c6211866d51277f1675fb/benchmark_runner/grafana/perf/dashboard.json)
I have pushed the changes, think its good enough for now